app/views/sessions/new.html.erb

```diff
@@ -1,5 +1,14 @@
 <%= form_tag session_path do %>
   <h3>Please Login</h3>
+
```

Comment:
This is probably going to appear on almost all pages, it might be worth breaking it out into a partial